Update template pages (#25)
Imageomics/Imageomics-guide: docs/wiki-guide/HF_ModelCard_Template_mkdocs.md

# Model Card Template

> Below is the Model Card Template for Imageomics and ABC. You can download or copy this content and paste it into a new Markdown file to create a new dataset card. 

Reviewer: ```suggestion
Below are the Model Card templates for Imageomics and ABC. You can download or copy the appropriate model card content and paste it into a new Markdown file to create a README for your model repo. 
```
